File: archive/crawler-log-folder.js
const minOperations = function(logs) {
  let ans = 0
  logs.forEach(el => {
    if (el === '../') {
      if (ans > 0) {ans--}
    } else if (el !== './') {
      ans++
    }
  })

  return ans
}

console.log(minOperations(["d1/","d2/","../","d21/","./"]))

File: go-archive/crawler-log-folder.go
func minOperations(logs []string) int {
    ans:=0

    for i:=0; i < len(logs); i++ {
        if (logs[i] == "../") {
            ans--
            if (ans < 0) {ans = 0}
        } else if (logs[i] == "./") {
            continue
        } else {
            ans++
        }
    }

    return ans
}